src/wrapped_radix_engine_toolkit/high_level/manifest_building/modify_manifest.rs

```diff
@@ -471,30 +455,8 @@ CALL_METHOD
                     ResourceAddress::sample(),
                     None
                 )
-            ]),
-            manifest
-        );
-    }
-
-    #[test]
-    #[should_panic(
-        expected = "Does not make sense to add more guarantees than there are instructions."
-    )]
-    fn test_modify_manifest_add_guarantees_panics_if_more_guarantees_than_instructions(
```

Comment:
Maybe we should keep this kind of test, but the assertion would be that it does not fail if there are lots of guarantees?

Reply:
I thought about it, but it seemed a bit strange to test that we _don't_ arbitrarily limit the number of guarantees... But I can of course add it.

Reply:
I agree, it is sort of a paranoia check, but still better to have